src/Service/InstallationService.php

```diff
@@ -1122,10 +1122,20 @@ private function createEndpoint(string $type, array $endpointData): ?Endpoint
         }
 
         $endpoint = $this->entityManager->getRepository('App:Endpoint')->findOneBy(['reference' => $endpointData['$id']]);
-        if ($endpoint !== null) {
+        if ($endpoint !== null && version_compare($endpointData['version'], $endpoint->getVersion()) <= 0) {
             $this->logger->debug('Endpoint found with reference '.$endpointData['$id']);
```

Comment:
Maybe add to this logger->debug:
", version number ('.$endpointData['version'].') is equal or lower than the current version"